### ISSUE TITLE
Shouldn't I be able to log in to other Mast sites with same U/P?

### ISSUE THREAD

**Issue by Mugsy323** [mastodon/mastodon#22421]
### Steps to reproduce the problem

1. I created an account on "Newsie.Social".
2. Tried to sign in to "Mastodon.social" using the same Username/Password used on "Newsie".
3. It says "No account found."

### Expected behaviour

Should have logged me in.

### Actual behaviour

Responds "No account found".

### Detailed description

When I signed up for "Newsie", it said since Mastodon is "decentralized" so I only need one account.

So why can't I sign it to the other site?

### Specifications

Not a browser issue. Same result with different browsers.

**Comment by connorshea** [comment]
This would require sharing the user's encrypted password *and* the encryption key to every other Mastodon instance, which isn't a good idea for various reasons. (Although, to be clear I believe mastodon uses bcrypt, so the key can only be used to encrypt a password, not to decrypt it, at least not without a massive server farm trying to brute force it).

**Comment by rbairwell** [comment]
Nope, You have an account (with that username and password) on newsie.social (a bit like having a username and password for Gmail). While you can communicate with Mastodon.social (or, to continue the email comparison - you can send and receive emails to Hotmail users from Gmail), you do not have an account on that system.

Like email, you could sign up for an entirely separate account on Mastodon.social (or any other server) if you wanted to separate things for whatever reason (again - in the email comparison - you probably have a personal email address and a work email address: they both reach "you" as a person and you can send and receive emails from anybody else, but they are technically separate).

**Comment by athosmoreno** [comment]
Each Mastodon instance is autonomous and independent of each other. You can follow and interact users from other instances (if our instance doesn't block them or isn't blocked by them) but your credentials are valid only for the instance that you signed up for.

**Comment by Mugsy323** [comment]
Thanks for the replies. Mastodon boasting about being "decentralized" and "needing only one account" is quite misleading. :(

**Comment by athosmoreno** [comment]
It is decentralized in the sense that you can interact with people outside of your instance. Since Mastodon is under the ActivityPub protocol you can reach people on instances running other ActivityPub compliant software (like Pleroma, Pixelfed, Pixelfed, etc). However, having one account working for all Mastodon instances would require a centralized authentication service which is not the point.
